File: mobile/src/main/java/com/alexstyl/specialdates/events/namedays/calendar/resource/NamedayJSON.java
package com.alexstyl.specialdates.events.namedays.calendar.resource;

import org.json.JSONArray;

class NamedayJSON {
    private final JSONArray data;
    private final JSONArray special;

    NamedayJSON(JSONArray data, JSONArray special) {
        this.data = data;
        this.special = special;
    }

    JSONArray getData() {
        return data;
    }

    JSONArray getSpecial() {
        return special;
    }
}

File: memento/src/main/java/com/alexstyl/specialdates/events/namedays/calendar/resource/NamedayJSON.kt
package com.alexstyl.specialdates.events.namedays.calendar.resource

import com.alexstyl.specialdates.events.namedays.NamedayLocale
import org.json.JSONArray

data class NamedayJSON(val locale: NamedayLocale, val namedaysJSON: JSONArray, val special: JSONArray)
